Docs/feature_memory_protection.md

```diff
@@ -284,7 +284,6 @@ Overhead is same as [Page Guards](#page-guards) and [Pool Guards](#pool-guards).
 
 - UefiPageGuard - Enable UEFI page guard
 - UefiPoolGuard - Enable UEFI pool guard
-- UefiFreedMemoryGuard - Enable Use-After-Free memory detection

```

Comment:
Should we update this doc to indicate that freed memory is RP by default?

Reply:
I'm going to do a separate PR for the full doc update but it's almost done